furusiya/model/keys/events.py

```diff
@@ -0,0 +1,24 @@
+from main_interface import Game
+from model.keys.key_callbacks import exit_to_main_menu_callback
+from model.systems.ai_system import AISystem
+
```

Comment:
"Events" makes me/people think event-driven design, which is not what this is; `callbacks` might be a better name.